protocol/synthetix/contracts/storage/CollateralConfiguration.sol

```diff
@@ -38,33 +46,61 @@ library CollateralConfiguration {
 
     /**
      * @dev Thrown when the amount being delegated is less than the minimum expected amount.
+     * @param minDelegation The current minimum for deposits and delegation set to this collateral type.
      */
     error InsufficientDelegation(uint256 minDelegation);
 
     struct Data {
-        /// must be true for depositing or collateral delegation
+        /**
+         * @dev Allows the owner to control deposits and delegation of collateral types.
+         */
         bool depositingEnabled;
-        /// accounts cannot mint sUSD if their debt brings their cratio below this value
+        /**
+         * @dev System-wide collateralization ratio for issuance of snxUSD.
+         * Accounts will not be able to mint snxUSD if they are below this issuance c-ratio.
+         */
         uint256 issuanceRatioD18;
-        /// accounts below the ratio specified here are immediately liquidated
+        /**
+         * @dev System-wide collateralization ratio for liquidations.
+         * Accounts below this c-ratio can will be immediately liquidated.
+         */
         uint256 liquidationRatioD18;
-        /// amount of token to award when an account is liquidated with this collateral type
+        /**
+         * @dev Amount of tokens to award when an account is liquidated.
+         */
         uint256 liquidationRewardD18;
-        /// address which reports the current price of the collateral
+        /**
+         * @dev The oracle manager node id which reports the current price for this collateral type.
+         */
         bytes32 oracleNodeId;
-        /// address which should be used for transferring this collateral
+        /**
+         * @dev The token address for this collateral type.
+         */
         address tokenAddress;
-        /// minimum delegation amount (other than 0), to prevent sybil/attacks on the system due to new entries.
+        /**
+         * @dev Minimum amount that accounts can delegate to pools.
+         * Helps prevent spamming on the system.
+         * Note: If zero, no minimum amount will be used.
```

Comment:
I thought we use the liquidation reward amount if this is less than it?

Reply:
And you are right! Fixed 🙏